Task1/transportation.js

```diff
@@ -0,0 +1,13 @@
+//4 -http://www.codewars.com/kata/transportation-on-vacation
+function rentalCarCost(d) {
+  var noOfDays = d;
```

Comment:
No need to reassign the parameter,with a variable in function,
you can write as:

function rentalCarCost(noOfDays){
...
}